src/client/sandbox/cookie.js

```diff
@@ -67,7 +67,7 @@ export default class CookieSandbox extends SandboxBase {
 
         //NOTE: all TestCafe sessions has same domain, so we should validate Domain attribute manually
```

Comment:
all TestCafe sessions have the same domain, so we should validate the Domain attribute manually